chirp/drivers/kguv980p.py

```diff
@@ -673,6 +687,22 @@ def _str_encode(in_str):
     return out_str
 
 
+def _str_encode_950(in_str):
+    out_str = ''
+    for c in in_str:
+        if ord(c) < 32 or ord(c) > 126:
+            out_str += ''
+        elif ord(c) >= 32 and ord(c) <= 47:
+            out_str += chr(int(ord(c) + 48))
+        else:
+            out_str += chr(int(ord(c) - 48))
+    while len(out_str) < 8:
+        out_str += chr(0x50)
+    if out_str == "        " or out_str == "":
```

Comment:
And this can be:
```
if not out_str.strip():
```
The next person that comes along will have to count how many spaces in that string trigger this special behavior, but the above makes it clear that it's "if it's all spaces, use this special thing".